### ISSUE TITLE
Plan how to make use of ground truth labels for training/val sets

### ISSUE THREAD

**Issue by kutayeroglu** [kutayeroglu/ijepa#26]
Should inspect 
`/stratch/dataset/imagenet-object-localization-challenge/ILSVRC/ImageSets/CLS-LOC$ cat train_cls.txt`
and 
`/stratch/dataset/imagenet-object-localization-challenge/ILSVRC/ImageSets/CLS-LOC$ cat val.txt`

Then, check linear probing script.

**Comment by kutayeroglu** [comment]
tail of `cat train_cls.txt`
> n15075141/n15075141_9933 1281164
> n15075141/n15075141_9942 1281165
> n15075141/n15075141_9993 1281166
> n15075141/n15075141_999 1281167

tail of `cat val.txt` 
> ILSVRC2012_val_00049996 49996
> ILSVRC2012_val_00049997 49997
> ILSVRC2012_val_00049998 49998
> ILSVRC2012_val_00049999 49999

**Comment by kutayeroglu** [comment]
Currently labels are read with 
```python
 train_dataset = datasets.ImageFolder(
     os.path.join(data_dir, "train"), transform=train_transform
 )
 
 val_dataset = datasets.ImageFolder(
     os.path.join(data_dir, "val"), transform=val_transform
 )
```

ImageFolder reads labels from the directory structure:
```
~/datasets/in1k/
├── train/
│   ├── n01440764/    # Class 0
│   │   ├── image1.jpg
│   │   └── image2.jpg
│   ├── n01443537/    # Class 1
│   │   └── image3.jpg
│   └── ...
└── val/
    ├── n01440764/    # Class 0
    └── ...
```
